### PR TITLE
Admission controller: enable DaemonSet admitter

### PR DESCRIPTION
**Pull request by aermakov-zalando**
A basic sanity check for reserved resources. Defaults will be updated after testing it.

### CODE REVIEW

File: cluster/config-defaults.yaml [hunk 1]
```diff
@@ -168,6 +168,12 @@ teapot_admission_controller_deployment_default_max_surge: "5%"
 teapot_admission_controller_deployment_default_max_unavailable: "1"
 teapot_admission_controller_inject_aws_waiter: "true"
 
+teapot_admission_controller_check_daemonset_resources: "true"
```
Comment (linki) [inline cluster/config-defaults.yaml]: Do you want to immediately turn it on?

Comment (aermakov-zalando) [reply cluster/config-defaults.yaml]: We'd see how it work in `dev`, but yeah, we could also be on the safe side.

**Comment by linki** [comment]
:+1:

**Comment by aermakov-zalando** [comment]
👍

**Pull request merged**